libs/ui/src/radio_group.tsx

```diff
@@ -121,6 +125,14 @@ export function RadioGroupWithRef<T extends RadioGroupValue>(
     value,
   } = props;
 
+  // The native radio input is currently incompatible with both BMD accessible
+  // controllers' custom navigation and the screen reader, so we drop it in
+  // voting machine UIs (currently identified by touch-specific themes) and
+  // rely solely on the button to trigger option selection.
+  const shouldUseNativeRadioInteraction = !isTouchscreen(
+    useCurrentTheme().screenType
+  );
```

Comment:
Seems like a reasonable assumption for now, that touch-specific theme = voter-facing UI. Spent a bit of time trying to come up with a more reliable alternative and couldn't think of anything obvious